src/types.ts

```diff
@@ -118,3 +119,10 @@ export type ResourceManagerContext = {
   connection: Connection;
   apiEndpoint: string;
 };
+
+export type ResourceManagerContextV1 = {
```

Comment:
I believe we can say here
```
export type ResourceManagerContextV1 = ResourceManagerContext & {
  connection: ConnectionV1;
};
```
So it's more obvious what's the actual difference

Reply:
Nice, didn't realise we can override with this syntax. It is better for sure.